Framework/test/testTrendingTask.cxx

```diff
@@ -51,6 +52,25 @@ struct CleanupAtDestruction {
   std::function<void()> mCallback = nullptr;
 };
 
+// https://stackoverflow.com/questions/424104/can-i-access-private-members-from-outside-the-class-without-using-friends
+template <typename Accessor, typename Accessor::type Member>
+struct DeclareGlobalGet {
+  friend typename Accessor::type get(Accessor) { return Member; }
+};
+
+struct TrendingTaskReductorAccessor {
+  using type = std::unordered_map<std::string, std::unique_ptr<Reductor>> TrendingTask::*;
+  friend type get(TrendingTaskReductorAccessor);
+};
+
+struct ReductorConfigAccessor {
+  using type = CustomParameters Reductor::*;
+  friend type get(ReductorConfigAccessor);
+};
+
+template struct DeclareGlobalGet<TrendingTaskReductorAccessor, &TrendingTask::mReductors>;
+template struct DeclareGlobalGet<ReductorConfigAccessor, &Reductor::mCustomParameters>;
+
```

Comment:
I have to admit, I spent 2 hours trying to understand it and I did not manage. Not saying that the solution is bad though, but I need some learning.